torch/nn/modules/sparse.py

```diff
@@ -122,7 +122,7 @@ def extra_repr(self):
         return s.format(**self.__dict__)
 
     @classmethod
-    def from_pretrained(cls, embeddings, freeze=True):
+    def from_pretrained(cls, embeddings, freeze=True, sparse=False):
```

Comment:
this change wasn't reviewed either I presume. Docstring wasn't updated atleast

Reply:
Caused by D7784132